### PR TITLE
Add Pypi badge to README

### PR DESCRIPTION
**Pull request by kafonek**
There's a couple badges I like to have at the top of README's in my repos, thought I'd start with adding in a simple pypi link to see what you all thought of them. Happy to include more in this PR or leave them off the project, either way.

image

(`pydantic-core` for comparison as another Python/Rust hybrid repo on pypi -- *to be clear I'm not a contributor there*. The different style/text between the top and bottom picture is a difference in using badge.fury.io vs shield.io, I don't have a real preference between them)
image

### CODE REVIEW

**Review by davidbrochart — APPROVED** [review]
Thanks @kafonek, feel free to add more badges if you want.

**Review by Waidhoferj — APPROVED** [review]
Good idea!

**Comment by kafonek** [comment]
I'm all about small PR's, let's roll with this and I'll put up others as we go. FWIW I can't merge this or the `hatch` PR, if one of you would do the honors.

**Pull request merged**
